README.md

```diff
@@ -0,0 +1,60 @@
+# Student alcohol consumption
+
+## Dataset
+[Student Alcohol Consumption](https://www.kaggle.com/datasets/uciml/student-alcohol-consumption?select=student-mat.csv)
+
+This dataset contains information about students that may influence their academic performance.
+
+## The purpose of the prediction 
+To determine the student's final grades based on their alcohol consumption and the other characteristics listed below
+
+## Selected features for prediction
+
+- **age**: Student's age (numeric: from 15 to 22)
+- **reason**: Reason to choose this school (nominal: close to 'home', school 'reputation', 'course' preference or 'other')
+- **studytime**: Weekly study time (numeric: 1 - <2 hours, 2 - 2 to 5 hours, 3 - 5 to 10 hours, or 4 - >10 hours)
+- **failures**: Number of past class failures (numeric: n if 1<=n<3, else 4)
+- **famsup**: Family educational support (binary: yes or no)
+- **paid**: Extra paid classes within the course subject (Math or Portuguese) (binary: yes or no)
+- **activities**: Extra-curricular activities (binary: yes or no)
+- **higher**: Wants to take higher education (binary: yes or no)
+- **internet**: Internet access at home (binary: yes or no)
+- **romantic**: With a romantic relationship (binary: yes or no)
+- **famrel**: Quality of family relationships (numeric: from 1 - very bad to 5 - excellent)
+- **freetime**: Free time after school (numeric: from 1 - very low to 5 - very high)
+- **goout**: Going out with friends (numeric: from 1 - very low to 5 - very high)
+- **Dalc**: Workday alcohol consumption (numeric: from 1 - very low to 5 - very high)
+- **Walc**: Weekend alcohol consumption (numeric: from 1 - very low to 5 - very high)
+- **absences**: Number of school absences (numeric: from 0 to 93)
+
+## All features
```

Comment:
Please, move  ```All features``` higher than ```Selected```
Cut description from __selected__ to __all__

Reply:
Then I think the description of the features can be moved back to the general list